ext_mongodb.php

```diff
@@ -1,6 +1,12 @@
 <?hh
 namespace MongoDB\Driver;
 
+interface TypeWrapper
+{
+	function fromType(\MongoDB\BSON\Type $type);
+	function toType() : \MongoDB\BSON\Type;
```

Comment:
Yes, the should be public.

I don't understand the:

| Any reason not to use MongoDB\BSON\Type at the top of the file?